src/features/discord/functions/add-discord-dev-prefix.ts

```diff
@@ -1,3 +1,6 @@
-export function addDiscordDevPrefix(message: Readonly<string>): string {
-  return `**[dev]** ${message}`;
+export function addDiscordDevPrefix(
+  name: Readonly<string>,
```

Comment:
If you are ok with having this name as optional then it should be the second parameter and you should definitely handle the case when this is nil or empty.
Note: I think that the parameter should be required nonetheless like you did